src/hotspot/share/opto/machnode.hpp

```diff
@@ -434,7 +434,7 @@ class MachConstantBaseNode : public MachIdealNode {
 
   virtual void emit(CodeBuffer& cbuf, PhaseRegAlloc* ra_) const;
   virtual uint size(PhaseRegAlloc* ra_) const;
-  virtual bool pinned() const { return UseRDPCForConstantTableBase; }
+  virtual bool pinned() const { return false; }
```

Comment:
The method can be removed because `Node::pinned` returns false.

Reply:
thank you to the head-up.  
confirmed that MachConstandBaseNode uses Node::pinned() const {return false; }, so it's safe to remove it. 

I will update it in next revision along with the obsolete flag.